src/webgpu/shader/execution/shader_io/fragment_builtins.spec.ts

```diff
@@ -1210,3 +1236,92 @@ g.test('inputs,front_facing')
       })
     );
   });
+
+g.test('inputs,sample_mask')
+  .desc(
+    `
+    Test fragment shader builtin(sample_mask) values.
+
+    Draws various triangles that should trigger different sample_mask values.
+    Checks that sample_mask matches what's expected.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('nearFar', [[0, 1] as const, [0.25, 0.75] as const] as const)
+      .combine('sampleCount', [1, 4] as const)
+      .combine('interpolation', [
```

Comment:
In this test the sample_mask value does not depend on the interpolation. So shouldn't we only set on kind of interpolation, to reduce test redundancy?

Reply:
It does depend on interpolation because `sample` turns on per-sample rendering. I can remove all but 2

Reply:
done

Reply:
ah, right. Thanks